```
[ERROR] Failed to execute goal com.yahoo.vespa:abi-check-plugin:8.514.6:abicheck (default) on project config-model-api: ABI spec mismatch.
[ERROR] To update run 'mvn package -Dabicheck.writeSpec'
[ERROR] -> [Help 1]
[ERROR]
```

--- a/config-model-api/abi-spec.json
+++ b/config-model-api/abi-spec.json
@@ -1340,7 +1340,8 @@
       "public long zookeeperPreAllocSize()",
       "public int documentV1QueueSize()",
       "public int maxContentNodeMaintenanceOpConcurrency()",
-      "public int maxDistributorDocumentOperationSizeMib()"
+      "public int maxDistributorDocumentOperationSizeMib()",
+      "public java.lang.Object sidecarsForTest()"
     ],
     "fields" : [ ]
   },

--- a/flags/src/main/java/com/yahoo/vespa/flags/custom/Sidecar.java
+++ b/flags/src/main/java/com/yahoo/vespa/flags/custom/Sidecar.java
@@ -1,3 +1,4 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.flags.custom;
 
 import com.fasterxml.jackson.annotation.JsonCreator;

--- a/flags/src/main/java/com/yahoo/vespa/flags/custom/SidecarImage.java
+++ b/flags/src/main/java/com/yahoo/vespa/flags/custom/SidecarImage.java
@@ -1,3 +1,4 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.flags.custom;
 
 import com.fasterxml.jackson.annotation.JsonCreator;

--- a/flags/src/main/java/com/yahoo/vespa/flags/custom/SidecarQuota.java
+++ b/flags/src/main/java/com/yahoo/vespa/flags/custom/SidecarQuota.java
@@ -1,3 +1,4 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.flags.custom;
 
 import com.fasterxml.jackson.annotation.JsonCreator;

--- a/flags/src/main/java/com/yahoo/vespa/flags/custom/Sidecars.java
+++ b/flags/src/main/java/com/yahoo/vespa/flags/custom/Sidecars.java
@@ -1,3 +1,4 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.flags.custom;
 
 import com.fasterxml.jackson.annotation.JsonCreator;

